tests/helpers.py

```diff
@@ -33,16 +33,16 @@
             "frames": [
                 {
                     "lidar": {
-                        "url": TEST_POINTCLOUD_URLS[0],
-                        "type": "pointcloud",
+                        "pointcloud_url": TEST_POINTCLOUD_URLS[0],
                         "reference_id": "lidar_frame_1",
                         "metadata": {},
                     },
```

Comment:
Can we make this frame a bit more complex - i.e. one lidar pointcloud and one image with camera_params? I think it's important to test the image upload functionality in the integration test as well